d2/src/main/java/com/linkedin/d2/jmx/XdsClientJmx.java

```diff
@@ -128,6 +150,26 @@ public void incrementReconnectionCount()
     _reconnectionCount.incrementAndGet();
   }
 
+  public void incrementRequestSentCount()
+  {
+    _resquestSentCount.incrementAndGet();
+  }
+
+  public void addToIrvSentCount(int delta)
```

Comment:
Hude nit, but this should match the other function names and be called `incrementIrvSentCount`

Reply:
not really, incrementXXX methods only increment the counter by 1.

Reply:
You can increment by any delta, the default happens to be one. It's a weird signature, but it's not a deal breaker